ELECTIONS.md

```diff
@@ -0,0 +1,201 @@
+- Announcing results and certifying the election
+
+## Voter Eligibility
+
+### Primary Eligibility Criteria
+The first voting of new member elections will be held by the TSC. This is to get through the process successfully for its first voting iteration, and to ensure goals of electing new members is completed in a timely manner. This will be amended in the future.
+
+## Candidate Eligibility
+
+### Basic Requirements
+To be eligible as a candidate, individuals must:
+- Be an active Project Maintainer or Contributor as defined in the [OpenSearch Project Technical Charter](https://opensearch.org/wp-content/uploads/2025/02/OpenSearch-Project-Technical-Charter-Final-9-13-2024.docx.pdf) (section 2e)
+- Have demonstrated significant technical leadership within the OpenSearch Project
+- Commit to serving the full term if elected
+
+### Nomination Process
+Candidates may:
+- Self-nominate by submitting a nomination form during the nomination period
+- Be nominated by any other eligible voter (with candidate's consent)
+- Be a current TSC member running for re-election
+
+### Candidate Materials
+Each new candidate must provide:
+- A candidate statement (maximum 500 words) describing:
+  - Their technical contributions to the OpenSearch Project as defined in the [OpenSearch Project Technical Charter](https://opensearch.org/wp-content/uploads/2025/02/OpenSearch-Project-Technical-Charter-Final-9-13-2024.docx.pdf) (section 2e)
+  - Their vision for the project's technical direction
+  - Specific areas they would focus on as a TSC member
+  - Their commitment to serving the full term
+- A brief biography highlighting relevant experience
+- Links to significant technical contributions, MAINTAINER docs they are part of, or technical proposals
+
+Each candidate seeking re-election must provide:
+- A note on the candidate nomination issue stating 're-election'
+- Their commitment to serving the full term
+
+*Candidate entries that do not contain all relevant information will be excluded from vote*
+
+## Voting Procedure
+
+### Voting Method
+Elections will use a **Condorcet ranking method with Instant Runoff Voting (IRV)** for tiebreaking. This method ensures that:
+- Voters rank candidates in order of preference
+- The candidate with the strongest overall support wins
+- Strategic voting is minimized
+
+*Voting mechanisms will be handled within the LFX Project Control Center tooling by the facilitator*
+
+### Ballot Distribution
+Election Facilitator will distribute secure electronic ballots to all eligible voters using a system that ensures:
+- Ballot secrecy and voter privacy
+- Prevention of duplicate voting
+- Auditability of results
+- Accessibility for all eligible voters
+
+### Voting Period
+- Minimum voting period: During a scheduled TSC meeting
+- Voting period will be clearly communicated and may be extended only for technical issues
+
+### Vote Counting
+The Election Facilitator will:
+- Use the Condorcet method to determine winners
```

Comment:
nit: instead of calling them winners, I'd use `elected candidates` or `electees`

Reply:
done